src/analyses/mCPRegistry.ml

```diff
@@ -139,7 +139,6 @@ struct
 
   open DLSpec
   open List
-  open Obj
 
   type t = (int * unknown) list
```

Comment:
I think we should get rid of `type unknown = Obj.t` while we're at it. This `unknown` shows up in all the MCP type signatures and obfuscates the fact that the unsafe `Obj` module is being used.